examples/2016-10-31/hello_world_vpc/README.md

```diff
@@ -0,0 +1,55 @@
+
+# Lambda function with VPC Access
+
+This example shows you how to create a Lambda function in VPC with the appropriate permissions using SAM. It primary aims to demonstrate Cloudformation parameters as well as a simplified configuration made possible with SAM Policies, therefore it'll not utilise API Gateway or any other Event source and as a result only the account owner can invoke it.
```

Comment:
"It primary" => "It primarily"

Reply:
"Lambda function in VPC" => "Lambda function in a VPC"?

Reply:
done!